apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MigratorApp.java

```diff
@@ -30,14 +32,27 @@
   /** This {@link System#exit(int)} value should be used when the migrations fail for any reason. */
   static final int EXIT_CODE_FAILED_MIGRATION = 2;
 
+  /**
+   * This {@link System#exit(int)} value should be used when hibernate validations fail for any
+   * reason.
+   */
+  static final int EXIT_CODE_FAILED_HIBERNATE_VALIDATION = 3;
+
   static final int EXIT_CODE_SUCCESS = 0;
 
   /**
-   * This method is called when the application is launched from the command line. Currently, this
-   * is a notional placeholder application that produces NDJSON-formatted log output.
+   * The list of packages to scan when doing hibernate validation; these packages should contain the
+   * database models that hibernate should attempt to validate. These are allowed to be from
+   * external packages. This may be better served coming from an application configuration.
+   */
+  static final List<String> hibernateValidationModelPackages =
+      Arrays.asList("gov.cms.bfd.model.rda", "gov.cms.bfd.model.rif");
```

Comment:
I don't think we need a mutable list here. Can we use List.of() instead?

Reply:
Fixed to use List.of!